app/components/support_interface/audit_trail_component.rb

```diff
@@ -19,7 +20,9 @@ def audits
         audits = audits.where(auditable_type: params[:auditable_type])
       end
 
-      audits.includes(:user).order('created_at desc').page(params[:page] || 1).per(60)
+      audits = audits.includes(:user).order(created_at: :desc)
+
+      pagy(audits, page: params[:page], items: 60)
```

Comment:
This should all happen at the controller level but to avoid changing loads of other things I'm ok with this staying here "for now"™️